### DISCUSSION TITLE
color-modes.js

### DISCUSSION THREAD

**Discussion by jweaver0312** [twbs/bootstrap#39129] [Q&A]
Using the Bootstrap provided sign in page example, alongside with their color-modes.js, is there a reason it would not work only when the site is routed through Cloudflare? If I expose the origin, it works fine, but once Cloudflare is turned back on, it gets borked and stops functioning altogether, leading to the flicker, and the manual auto/light/dark toggle not working at all. Even after uninstalling all Cloudflare apps which moved the color-modes.js script up to the top as the first thing under \<head\>, it still does not want to function. I'm not seeing any reason it wouldn't function.

**Comment by OMEGAYALFA** [comment]
aun sigue con este problema la v.5.3 y no se ha podido solucionar...
lo extraño es que en ordenador funciona bien y mientras en el teléfono no funciona, pero seguro que es problemas de la cloudflare, por q  he subido la web en otro dominio que no utiliza cloudflare y va bien el modo oscuro.... ni desactivándo la cloudflare me funciona;; :(
